keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -207,6 +231,16 @@ Since falling back to etcd won't work, we should fail the requests and rely on
 rate limiting to prevent cascading failure.  I.e. `Retry-After` HTTP header (for
 well behaved clients) and [Priority and Fairness](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md).
 
+For such situations we will provide a dedicated metric to detect a problem,
+and a per-request override for debugging. If there is a ongoing problem,
+user can reconfigure kube-apiserver to always pass request to etcd by 
+disabling the feature flag. Metric `apiserver_watch_cache_read_wait`
+will measure wait time experienced by reads for watch cache to become fresh.
+Per request override will be done via a HTTP request header `k8s.io/apiserver/skip-watch-cache`.
+
+Having a manual explicit fallback rather than automatic implicit, gives users
+full control over and ensures accurate APF cost estimates.
```

Comment:
Please elaborate more on that - especially why the second one is important one.

Reply:
Done.